howto/android/system-tracing.md

```diff
@@ -0,0 +1,48 @@
+This guide describes how to enable system tracing in Android using Anbox Cloud. 
+
+## Enable Android System Tracing
+
+1. To enable Android system tracing, add the `android.enable_tracing` flag into the feature file. In the Android container, run:
+
+   ```bash
+   echo "android.enable_tracing" >> /var/lib/anbox/features
+    ```
+ 2. Specify the category you want to trace in the Android system using the `debug.atrace.tags.enableflags` system property. The supported categories are defined in [trace.h](https://cs.android.com/android/platform/superproject/main/+/main:system/core/libcutils/include/cutils/trace.h;l=50?q=ATRACE_TAG_INPUT). 
+ 
+    For instance, to enable the graphics category, add the following to the `/var/lib/anbox/android.json` file:
+
+    ```json
+      {
+        "hardware": "ranchu",
+        "api_level": 31,
+        "properties": [ "debug.atrace.tags.enableflags=2" ]
+      }
+    ```
```

Comment:
I think we need to reboot the instance after changing the android.json file, but we need to double-check this with @morphis or @adglkh.